Backbase/variants
# Add PostSwitchScript to iOSConfiguration and to iOSVariant

### What does this PR do
- in this PR, post switch script is added, so whenever a user wants to run a specific script after switching variants, they can either do that for a specific variant, or for global switching, or for both
- if the user specifies both variant specific and global postSwitchScript, we will run the variants specific one first and then the global one

### How can it be tested
add a postSwitchScript to your variants.yml and it will be executed
- Add one postSwitchScript in a variant
- Add another one on the global layer of the yaml file
- Remove both to make sure that it's optional

### Task
resolves #X (where `#X` is the number of an issue)

### Checklist:
